2017_01_04_word_count/wc.js

```diff
@@ -0,0 +1,23 @@
+// Run this with:
+// node wc.js quotes.txt
+var fileName = process.argv[2];
+fs = require('fs')
+fs.readFile(fileName, 'utf8', function (err, data) {
+    if (err) {
+        return console.log(err);
+    }
+
+    console.log(lineCount(data) + ' ' + wordCount(data) + ' ' + totalChars(data) + ' ' + fileName);
+});
+
+function lineCount(data) {
```

Comment:
So as a general style thing here, a good force of habit for times it does matter is putting functions before where they are called.  So I'd expect your readFile stuff to be at the bottom.